fire/__main__.py

```diff
@@ -0,0 +1,16 @@
+"""
```

Comment:
Sorry to be so picky; have to satisfy our internal linter.
Docstring/header format should be one of:

```
"""One line docstring.

Rest of docstring.
"""

from __future__ import absolute_import
from __future__ import division
from __future__ import print_function

```
or
```
"""One line docstring."""

from __future__ import absolute_import
from __future__ import division
from __future__ import print_function

```